### PR TITLE
validate

### PR DESCRIPTION
**Pull request by asadmisrani**
The script.js file was modified.
When the user submits the form, the email is in lowercase.
If the validation is OK, the form is sent.
If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the form is not sent.

### CODE REVIEW

File: script.js
Comment (shella12) [inline script.js:1]: - [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#project-requirements) , when the user submits the form, you should check if the email is in lower case `i.e. email === email.toLowerCase()`. User shouldn't be able to submit the form if email is not lower case. Please make sure to add this validation to your contact form.

**Review by shella12 — CHANGES_REQUESTED** [review]
Hi @asadmisrani and @shm04 , 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Professional README.md ✅   
- Great UI Implementation ✅ 

## Required Changes ♻️

- [ ] Please make sure to set up the linters for `javascript ` i.e. `ESlint`. Use [this](https://github.com/microverseinc/linters-config/tree/master/javascript) repo link for setting up the linters for javascript projects .

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js
Comment (asadmisrani) [reply script.js:1]: The liners for javascript i.e. ESlint created.
when the user submits the form User shouldn't be able to submit the form if the email is not lowercase.
The validation has been added in contact form.

Comment (asadmisrani) [reply script.js:1]: The liners for javascript i.e. ESlint created.
when the user submits the form User shouldn't be able to submit the form if the email is not lowercase.
The validation has been added to the contact form.

Comment (asadmisrani) [reply script.js:1]: The liners for javascript i.e. ESlint created.
when the user submits the form User shouldn't be able to submit the form if the email is not lowercase.
The validation has been added to the contact form.

**Review by Strangeal — COMMENTED** [review]
Hi @asadmisrani, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have Eslint setup. Kindly add [eslint](https://github.com/microverseinc/linters-config/tree/master/html-css-js) to your project and resubmit for review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by joel-kalema — APPROVED** [review]
Hi Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asadmisrani** [comment]
Thanks a lot.

**Comment by nicupop729** [comment]
@joel-kalema please note that the CRR was to due to be invalidated because the linters.yml file is not having the linters for the JS code.

**Pull request merged**